common/tsconfig.json

```diff
@@ -1,20 +1,30 @@
 {
-  "$schema": "https://json.schemastore.org/tsconfig",
+  "$schema": "https://schemastore.org/tsconfig",
 
   "include": ["src/**/*"],
 
   "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "./dist",
```

Comment:
1. do we need `"noEmit": true` ?
2. I don't see any direct usages of `tsc` - how do we compile/check TS for errors?

Reply:
The build and checks are part of the rollup build: https://github.com/rollup/plugins/tree/master/packages/typescript